tools/resource-gen/main.go

```diff
@@ -184,32 +304,36 @@ func SelectorsForMessage(m protoreflect.MessageDescriptor) []string {
 }
 
 type ResourceInfo struct {
-	ResourceName     string
-	ResourceType     string
-	ProtoType        string
-	Selectors        []string
-	SkipRegistration bool
-	SkipValidation   bool
-	Global           bool
-	KumactlSingular  string
-	KumactlPlural    string
-	WsReadOnly       bool
-	WsAdminOnly      bool
-	WsPath           string
-	KdsDirection     string
+	ResourceName       string
+	ResourceType       string
+	ProtoType          string
+	Selectors          []string
+	SkipRegistration   bool
+	SkipValidation     bool
+	SkipNativeGenerate bool
+	ScopeNamespace     bool
+	Global             bool
```

Comment:
As @jpeach pointed out Global and ScopeNamespace are the same thing I believe.

Reply:
I tried that originally, and only added the extra option when results didn't match. Eg Dataplane insight is ScopeNamespace, but not Global protobuf option.

Reply:
That's weird :) Could it be a bug?

Reply:
Actually I'm talking crap these are different :)